runtime/moonbeam/src/lib.rs

```diff
@@ -629,7 +629,18 @@ parameter_types! {
 }
 
 /// The type used to represent the kinds of proxying allowed.
-#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug)]
+#[derive(
+	Copy,
+	Clone,
+	Eq,
+	PartialEq,
+	Ord,
+	PartialOrd,
+	Encode,
+	Decode,
+	Debug,
+	max_encoded_len::MaxEncodedLen,
```

Comment:
What does this do? (Asking for my own learning)

Reply:
https://crates.parity.io/max_encoded_len/trait.MaxEncodedLen.html
> Items implementing MaxEncodedLen have a statically known maximum encoded size.